Zend/zend_API.h

```diff
@@ -714,7 +714,9 @@ static zend_always_inline zend_result zend_forbid_dynamic_call(void)
 	ZEND_ASSERT(ex != NULL && ex->func != NULL);
 
 	if (ZEND_CALL_INFO(ex) & ZEND_CALL_DYNAMIC) {
-		zend_throw_error(NULL, "Cannot call %s() dynamically", get_active_function_name());
```

Comment:
can you please add a test for this in ext/zend_test? Otherwise, the change looks sensible to me.

Reply:
Thanks for the review! I will finish it today.